src/app/data-client.ts

```diff
@@ -27,6 +27,50 @@ export class DataClient {
     this.service = new DataServiceClient(serviceHost, grpcOptions);
   }
 
+  /**
+   * Obtain unified tabular data and metadata, queried with SQL.
+   *
+   * @param organizationId The ID of the organization that owns the data
+   * @param query The SQL query to run
+   * @returns An array of data objects
+   */
+  async tabularDataBySQL(organizationId: string, query: string) {
+    const { service } = this;
+
+    const req = new pb.TabularDataBySQLRequest();
+    req.setOrganizationId(organizationId);
+    req.setSqlQuery(query);
+
+    const response = await promisify<
```

Comment:
Does this already handle if a gRPC error is returned?

Reply:
The user will have to handle any gRPC errors using `try/catch`